crypto/s2n_rsa.c

```diff
@@ -29,6 +29,15 @@
 #include "utils/s2n_safety.h"
 #include "utils/s2n_blob.h"
 
+int s2n_rsa_public_key_set_from_openssl(struct s2n_rsa_public_key *s2n_rsa, RSA *openssl_rsa)
```

Comment:
`s2n_rsa_public_key` is opaque, but applications that use the Cert Verification Callback are expected to set the `s2n_rsa_public_key` with the public key in the leaf Cert, so they need a way to set `s2n_rsa_public_key`. This API allows them to set the `s2n_rsa_public_key` from a given OpenSSL RSA Public key.